Donation-platform/src/main/java/com/google/servlets/TagScoreUpdateServlet.java

```diff
@@ -56,17 +58,25 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
         DatastoreService ds = DatastoreServiceFactory.getDatastoreService();
+        DbCalls dbCalls = new DbCalls(ds);
         FindTrendingCharities findTrending = new FindTrendingCharities(ds);
 
-        Collection<Tag> tagsToUpdate = new ArrayList<>();
-        Enumeration attributeNames = request.getAttributeNames();
-        while (attributeNames.hasMoreElements()) {
-            tagsToUpdate.add((Tag) attributeNames.nextElement());
-        }
+        String outputmsg;
```

Comment:
Nitpick: consistent variable naming case. outputmsg is lowercase while others camelCase.

Reply:
fixed